### ISSUE TITLE
Add new check: Check if all Elasticache Replication Group encryption parameters are enabled and set

### ISSUE THREAD

**Issue by nicolasbelanger** [bridgecrewio/checkov#43]
It would be nice to validate if all encryption features for AWS Elasticache Replication Group are enabled.  

From https://www.terraform.io/docs/providers/aws/r/elasticache_replication_group.html:

```
- at_rest_encryption_enabled - (Optional) Whether to enable encryption at rest.
- transit_encryption_enabled - (Optional) Whether to enable encryption in transit.
- auth_token - (Optional) The password used to access a password protected server. Can be specified only if transit_encryption_enabled = true.
- kms_key_id - (Optional) The ARN of the key that you wish to use if encrypting at rest. If not supplied, uses service managed encryption. Can be specified only if at_rest_encryption_enabled = true. 
```

**Comment by schosterbarak** [comment]
Hey @nicolasbelanger, Thank you for your feedback! Indeed a nice check to add :)

we would work on a guide of how to contribute a new check and would love you to contribute it. WDYT?

**Comment by schosterbarak** [comment]
@nicolasbelanger  take a look:
https://bridgecrewio.github.io/checkov/5.Contribution/New-Check.html

**Comment by schosterbarak** [comment]
@nicolasbelanger done: https://github.com/bridgecrewio/checkov/pull/55 . new checks are available at `checkov==1.0.116`
